src/app/modules/navigator/test-exec-navigator.component.ts

```diff
@@ -121,20 +131,38 @@ export class TestExecNavigatorComponent implements OnInit, OnDestroy {
 
   handleCancelRequest(): void {
     // TODO: to be implemented
+    this.log('ERROR: cancel running test not implemented yet');
+  }
+
+  buildUITestRunFromSingleTest(tclPath: string, executingTestResourceUrl: string, date: Date): UITestRun {
+    const shortenedTestPath = tclPath.substring(14, tclPath.length - 4); // cut off leading 'src/test/java/' and trailing '.tcl'
```

Comment:
takes a lot of things on faith, like the path actually starting with 'src/test/java/'… In general, maybe we should have the path prefix configurable somewhere? I guess right now its also hardwired in the `TreeFilterService`…